### ISSUE TITLE
Polygon not being extruded properly

### ISSUE THREAD

**Issue by ralphstodomingo** [uber/deck.gl#802]
I tried recreating the GeoJsonLayer inside our app, and got this instead of what was portrayed in the demo:
![image](https://user-images.githubusercontent.com/15341353/28505697-cb6c6fd8-7058-11e7-879b-f2b0dd7fc36d.png)

Saw inside [website/src/components/demos/layer-demo-base.js](https://github.com/uber/deck.gl/blob/master/website/src/components/demos/layer-demo-base.js) that upon initialization of `deck.gl`, something like this was being called:
```es6
setParameters(gl, {
   depthTest: true,
   depthFunc: gl.LEQUAL,
 });
```
The 4.0 release branch contains something like this:
```es6
gl.enable(gl.DEPTH_TEST);
gl.depthFunc(gl.LEQUAL);
```
Can I ask what deck.gl uses this particular snippet for? Does every app using deck.gl need this? Can't seem to find any mention what this does in the docs.

**Comment by 1chandu** [comment]
The two code snippets you have above are functionally equal. The first one uses new luma API (which should be more effiencent), are the results different when using one over the other?

**Comment by ralphstodomingo** [comment]
@1chandu I ended up putting the second snippet because at `deck.gl@4.0.6` and `luma.gl@3.0.2` the first one returns an error that `setParameters` isn't a function, guessing it's on a version more recent than the one I'm using (which was given with `yarn install deck.gl luma.gl`).

I'd say this wasn't the expected behavior, though: had I not looked at the actual demo source code I might not have noticed the snippets, it isn't in this page either [GeoJsonLayer demo](http://uber.github.io/deck.gl/#/documentation/layer-catalog/geojson-layer). Might this be included in the docs somewhere or be enabled by default when using deck.gl?

Closing this, hope it helps someone stuck.
